vm/Cargo.toml

```diff
@@ -15,3 +15,9 @@ num-derive = "0.3.0"
 clock = { path = "../node/clock" }
 thiserror = "1.0.11"
 crypto = { path = "../crypto" }
+commcid = { path = "../utils/commcid" }
+
+[dependencies.filecoin-proofs-api]
```

Comment:
Why the departure of our standard way of declaring dependencies?

Reply:
Also, don't understand why we need to specify a version if we already specify a rev.

Reply:
just for safety, and the reason I used this format is just to make updating commit easier and more visible, I'll change to other format tho